### PR TITLE
Remove uneeded uses of antislash in testing

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

File: library/src/interactor_impl.cxx [hunk 1]
```diff
@@ -551,7 +553,7 @@ bool interactor_impl::playInteraction(const std::string& file)
     this->Internals->Recorder->SetInteractor(this->Internals->VTKInteractor);
 #endif
 
-    std::string cleanFile = vtksys::SystemTools::CollapseFullPath(file);
+    std::string cleanFile = vtksys::SystemTools::CollapseFullPath(outPath);
```
Comment (mwestphal) [inline library/src/interactor_impl.cxx:556]: we should rely on std::filesystem, not vtksys imo

File: library/testing/TestSDKInteractorCallBack.cxx [hunk 2]
```diff
@@ -31,11 +31,8 @@ int TestSDKInteractorCallBack(int argc, char* argv[])
 
   // Test callbacks
   std::string filename = "TestSDKInteractorCallBack";
-#if defined(_WIN32)
-  std::string interactionFilePath = std::string(argv[2]) + "..\\..\\" + filename + ".log";
-#else
+
   std::string interactionFilePath = std::string(argv[2]) + "../../" + filename + ".log";
```
Comment (mwestphal) [inline library/testing/TestSDKInteractorCallBack.cxx:35]: Is this actually needed ? Maybe it works out of the box ?

File: library/testing/TestSDKInteractorCallBack.cxx [hunk 3]
```diff
@@ -31,11 +31,8 @@ int TestSDKInteractorCallBack(int argc, char* argv[])
...
-#endif
```
Comment (mwestphal) [inline library/testing/TestSDKInteractorCallBack.cxx:38]: If it doesn't work, can you check that it works for --input, --ouput and --config ?

File: library/testing/TestSDKInteractorCallBack.cxx (hunk 2 above)
Comment (Meakk) [reply library/testing/TestSDKInteractorCallBack.cxx:35]: Actually yes, it works out of the box.

**Review by mwestphal — APPROVED** [review]

**Pull request merged**